keps/sig-windows/windows-runtimeclass-support.md

```diff
@@ -0,0 +1,555 @@
+
+- Example of a RuntimeClass that would restrict a pod to Windows Server 2019 / 1809.
+
+    ```yaml
+    apiVersion: node.k8s.io/v1beta1
+    kind: RuntimeClass
+    metadata:
+      name: windows-1809
+    handler: 'docker'
+    scheduling:
+      nodeSelector:
+        kubernetes.io/os: 'windows'
+        kubernetes.io/arch: 'amd64'
+        node.kubernetes.io/osversion: '10.0.17763'
+      tolerations:
+      - effect: NoSchedule
+        key: windows
+        operator: Equal
+        value: "true"
+    ```
+
+- Example of a RuntimeClass that would restrict a pod to Windows Server version 1903
+
+    ```yaml
+    apiVersion: node.k8s.io/v1beta1
+    kind: RuntimeClass
+    metadata:
+      name: windows-1903
+    handler: 'docker'
+    scheduling:
+      nodeSelector:
+        kubernetes.io/os: 'windows'
+        kubernetes.io/arch: 'amd64'
+        node.kubernetes.io/osversion: '10.0.18362'
+      tolerations:
+      - effect: NoSchedule
+        key: windows
+        operator: Equal
+        value: "true"
+    ```
+
+#### Story 2 - Forward compatibility with Hyper-V
+
+Once a new version of Windows Server is deployed using [Windows CRI-ContainerD], Hyper-V can be enabled to provide backwards compatibility and run Windows containers based on the previous OS version. Cluster admins can pick between two different approaches to move applications forward.
+
+1. Create a new RuntimeClass that Pods can use to try out Hyper-V isolation on the new version of Windows.
+
+    ```yaml
+    apiVersion: node.k8s.io/v1beta1
+    kind: RuntimeClass
+    metadata:
+      name: windows-1809-hyperv
+    handler: 'containerd-hyperv-17763'
+    scheduling:
+      nodeSelector:
+        kubernetes.io/os: 'windows'
+        kubernetes.io/arch: 'amd64'
+        node.kubernetes.io/osversion: '10.0.18362'
+      tolerations:
+      - effect: NoSchedule
+        key: windows
+        operator: Equal
+        value: "true"
+    ```
+
+1. Once sufficient testing is done, the RuntimeClass from Story 1 could be updated. This would cause new deployments to go to these nodes, without having to update them individually. The nodes running the previous Windows version could be drained and removed from the cluster. As the pods running on them are rescheduled, the new RuntimeClass will be applied.
+
+    ```yaml
+    apiVersion: node.k8s.io/v1beta1
+    kind: RuntimeClass
+    metadata:
+      name: windows-1809
+    handler: 'containerd-hyperv-17763'
+    scheduling:
+      nodeSelector:
+        kubernetes.io/os: 'windows'
+        kubernetes.io/arch: 'amd64'
+        node.kubernetes.io/osversion: '10.0.18362'
+      tolerations:
+      - effect: NoSchedule
+        key: windows
+        operator: Equal
+        value: "true"
+    ```
+
+Another RuntimeClass could still be run on the same hosts to use updated containers without Hyper-V isolation.
+
+    ```yaml
```

Comment:
Note: this isn't rendering properly (wrong indent?).

Reply:
thanks! fixed